src/manager.js

```diff
@@ -97,26 +92,31 @@ AdManager.prototype.initGoogleTag = function() {
   this.targeting = global.TARGETING || TargetingPairs.getTargetingPairs(AdZone.forcedAdZone()).pageOptions;
 
   this.setPageTargeting();
-  if (this.options.amazonEnabled) {
-    this.initAmazonA9();
-  }
 
   this.initialized = true;
 
   this.loadAds();
 };
 
 /**
- * initializes A9
+ * Fetch Amazon A9/Matchbuy/APS bids
  *
  * @returns undefined
 */
-AdManager.prototype.initAmazonA9 = function() {
-  if (typeof amznads !== "undefined" && amznads.apiReady) {
-    this.amznads = amznads;
-    this.amznads.getAds(this.amazonId);
-    this.amznads.setTargetingForGPTAsync('amznslots');
-  }
+AdManager.prototype.fetchAmazonBids = function(elementId, gptSizes) {
+	
+	window.apstag.fetchBids({
+		slots: [{
+			slotID: elementId,
+			sizes: gptSizes
+		}],
+		timeout: 2e3
+	}, function (bids) {
```

Comment:
We should change this to a named function so we can unit test separately.

Reply:
Hmm yeah that makes sense to potentially stub the bids, but i'm a little worried that we might break the a9 api.